moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -45,6 +45,8 @@
 #include <moveit/ompl_interface/constraints_library.h>
 #include <moveit/kinematic_constraints/utils.h>
 #include <moveit/profiler/profiler.h>
+#include <moveit/utils/lexical_casts.h>
+#include <eigen_conversions/eigen_msg.h>
```

Comment:
This extra include introduces a regression in `melodic-devel`: 
http://build.ros.org/view/Mbin_uB64/job/Mbin_uB64__moveit_planners_ompl__ubuntu_bionic_amd64__binary/17/console
We removed the `eigen_conversions` tf1 dependency in #830

Reply:
Argh! Can you device a fix?

Reply:
Yep! This is a spurious include. It's as simple as removing that line.